ecs-agent/logger/log.go

```diff
@@ -144,21 +160,31 @@ func reloadConfig() {
 }
 
 func seelogConfig() string {
+	platformLogConfig := platformLogConfig()
 	c := `
 <seelog type="asyncloop">
-	<outputs formatid="` + Config.outputFormat + `">
+	<outputs formatid="` + Config.outputFormat + `">`
+	if platformLogConfig != "" || Config.logToStdout {
+		c += `
 		<filter levels="` + getLevelList(Config.driverLevel) + `">
-			<console />`
-	c += platformLogConfig()
-	c += `
+		`
+		if Config.logToStdout {
+			c += `	<console />`
+		}
+		c += platformLogConfig
+		c += `
 		</filter>`
+	}
```

Comment:
If am finding this block a bit hard to read. Do you think the following is equivalent and reads better?
```go
stdoutFilterChildren := []string{}
if Config.logToStdout {
    stdoutFilterChildren = append(stdoutFilterChildren, ` <console />`)    
}
if platformLogConfig != "" {
    stdoutFilterChildren = append(stdoutFilterChildren, platformLogConfig)
}
if len(stdoutFilterChildren) > 0 {
    c += `
    <filter levels="` + getLevelList(Config.driverLevel) + `">
    `
    c += strings.Join(stdoutFilterChildren, "\n")
    c += `
    </filter>`
}
```

Reply:
ack, will change to this approach